packages/ui/src/components/EventsCard/EventsCard.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+import { Box, BoxProps } from '../Box';
+import { Button } from '../Button';
+import { Link } from '../Link';
+import { Typography } from '../Typography';
+import { BUTTON_TEXT, MAIN_TITLE } from './EventsCard.mocks';
+
+export type Event = {
+  id: number;
+  title: string;
+  date: Date | { from: Date; to: Date };
+  description: string;
+  url: string;
+};
+
+export type EventCardProps = { events: Event[] } & BoxProps;
+
+const addZeroToDateValue = (dateValue: number): string => {
+  if (dateValue === undefined) {
+    return '';
+  }
+
+  return dateValue >= 10 ? `${dateValue}` : `0${dateValue}`;
+};
+
+const getDayAndMonth = (date: Date): string => {
+  if (date === undefined) {
+    return '';
+  }
+
+  return `${addZeroToDateValue(date.getDate())}.${addZeroToDateValue(date.getMonth() + 1)}`;
+};
+
+const getHourAndMinute = (date: Date): string => {
+  if (date === undefined) {
+    return '';
+  }
+
+  return `${addZeroToDateValue(date.getHours())}:${addZeroToDateValue(date.getMinutes())}`;
+};
+
+export const EventsCard = ({ events, ...props }: EventCardProps) => {
+  return (
+    <Box width="100%" borderRadius={8} padding="32px" boxShadow="base" backgroundColor="white" {...props}>
+      <Typography size="xl" color="gray.900" fontWeight={800} lineHeight="28px" mb="24px">
```

Comment:
🤔 Typography shouldn't receive `fontWeight` (use `weight` instead) and `lineHeight` properties. I would need to look at why it was possible to pass them

Reply:
Hard to say, typescript show us a error ,,Type 'number' is not assignable to type 'undefined'' but storybook work well and text is bold.